openid-sharedsignals-framework-1_0.md

```diff
@@ -762,6 +764,12 @@ which can be used by Event Receivers to create and delete one or more Event Stre
 The API can also be used to query and update the Event Stream's configuration and status,
 add and remove Subjects, and trigger verification for those streams.
 
+Unless there exists some other method of establishing trust between a Transmitter and
+Receiver, all Stream Management API endpoints MUST use HTTP over TLS {{RFC2818}}
+and standard HTTP authentication and authorization schemes, as per {{RFC7235}}.
+The authorization MUST scope a Receiver to specific stream IDs, such that only that
```

Comment:
I'm not sure about a few things here:

- Using "scope" as a verb. Should we say "MUST restrict"? That raises the question how should it restrict. I'd suggest using language like "The authorization scheme used MUST be capable of restricting a stream to only one Receiver"
- The bigger point I'm unsure about is whether we should have the "MUST" language here at all. It is possible for a situation like: Receiver A has a stream for all "token-claims-change" events. Receiver B has a stream for "assurance level change" events. But both Receiver A and Receiver B share the stream for "session-revoked" event. They can both effect the required changes on receiving the event, so it doesn't matter to them which Receiver receives it.

Reply:
Both points make sense. I'll update the "scope" language and broaden the protection so that the Transmitter can use the auth to associate a Receiver with one or more streams without requiring the association be 1:1.